borderland-tool/borderland_tool/fcfs/first_come_first_served.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/python3
+import csv
+import random
+import json
+
+from datetime import datetime, timezone, timedelta
+from dateutil import parser
+
+
+class FCFS:
+    def __init__(self, pretix, csvfile, quota):
+        self.pretix = pretix
+        self.csvfile = csvfile
+        self.quota = quota
+        self.registered_without_membership_csv = self.load_csv()
+        self.has_order = self.has_voucher = None
+
+    def registrations_to_csv(self):
```

Comment:
also a lot of repetition here from the lottery class, but i guess in the short term it's safer.

Reply:
I'm aware, this started as a quick copy of the "lottery" command